packages/app/src/App.tsx

```diff
@@ -86,6 +86,8 @@ import {
   UserNotificationSettingsCard,
 } from '@backstage/plugin-notifications';
 
+import '@backstage/canon/css/styles.css';
```

Comment:
Just to discuss this a bit, how do you feel about importing here vs `src/index.tsx`?

Reply:
You're right it could be installed in `index.tsx`. I tried it and it works.